src/cargo-fuzz.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+use std::process;
+
+
+fn main() {
```

Comment:
Something like that, yeah!

On Sat, Apr 16, 2016 at 4:08 PM, Corey Farwell <notifications@github.com>
wrote:

> In src/cargo-fuzz.rs
> <https://github.com/frewsxcv/afl.rs/pull/46#discussion_r59973194>:
>
> > @@ -0,0 +1,15 @@
> > +use std::env;
> > +use std::process;
> > +
> > +
> > +fn main() {
>
> IMO cargo fuzz should also build the example file, like cargo run --example
>
> For example, if I was in a directory of a crate, and typed cargo fuzz,
> it'd build the crate and run afl-fuzz with the built binary as a
> parameter?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/frewsxcv/afl.rs/pull/46/files/11b380a8204a5d6e3ee513b41f0420e7dae2dc02#r59973194>
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6